### PR TITLE
Prototype: Activity log upgrade nudges

### PR DESCRIPTION
**Pull request by keoshi**
### This is a prototype, do not merge!

This PR contains some examples of how we can show upgrade nudges for the Activity log.

More info here: p9rlnk-iP-p2

### Jetpack

screen shot 2018-05-28 at 19 32 42

### WordPress.com Simple

screen shot 2018-05-28 at 19 31 44

### CODE REVIEW

File: client/my-sites/stats/activity-log/index.jsx [hunk 1]
```diff
@@ -300,6 +308,47 @@ class ActivityLog extends Component {
...
+							feature={ FEATURE_JETPACK_ESSENTIAL }
+							list={ [
+								'Changes made to your posts and pages',
+								'Historical record of site costumizations',
+								'Comments, pingbacks, and more',
+							] }
+							plan={ PLAN_PERSONAL }
+							title="See all your site activity"
+						/>
+					</div>
```
Comment (roccotripaldi) [inline client/my-sites/stats/activity-log/index.jsx]: We can work a little on the copy here. I would suggest:
"Upgrade your WordPress.com experience" for a title

For features, I'd list:
"Get a custom domain name"
"Remove WordPress.com ads"
"See 30 days of past activity"

And change the callToAction to "Upgrade"

File: client/my-sites/stats/activity-log/activity-log-demo.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActivityIcon from '../activity-log-item/activity-icon';
+import Gravatar from 'components/gravatar';
+
+class ActivityLogDemo extends Component {
```
[the remark this answers is not in the document]
Comment (keoshi) [reply client/my-sites/stats/activity-log/activity-log-demo.jsx]: Yeah, I tried to do that, but couldn't. :D

If we can do that without too much ado that would be wonderful!

File: client/my-sites/stats/activity-log/index.jsx [hunk 3]
```diff
@@ -300,6 +308,47 @@ class ActivityLog extends Component {
 		);
 	}
 
+	renderUpgradeBanner() {
+		return (
```
[the remark this answers is not in the document]
Comment (keoshi) [reply client/my-sites/stats/activity-log/index.jsx]: Definitely. I didn't implement any logic, this PR was meant as a live prototype only.

**Comment by simison** [comment]
Follow-ups: https://github.com/Automattic/wp-calypso/pull/25813

**Pull request closed without merging**